frontend/javascripts/oxalis/model/accessors/tool_accessor.ts

```diff
@@ -155,6 +154,27 @@ function _getSkeletonToolInfo(
 }
 const getSkeletonToolInfo = memoizeOne(_getSkeletonToolInfo);
 
+function _getBoundingBoxToolInfo(hasSkeleton: boolean, isSkeletonLayerTransformed: boolean) {
+  if (isSkeletonLayerTransformed) {
+    return {
+      [AnnotationToolEnum.BOUNDING_BOX]: {
+        isDisabled: true,
+        explanation: hasSkeleton
+          ? "The bounding box tool is disabled because the bounding boxes are rendered transformed like the skeleton layer. Use the left sidebar to render them without transformations by letting the skeleton layer render without any transformations."
```

Comment:
I tried to improve the text a bit. maybe like this?

```suggestion
          ? "The bounding box tool is disabled because the bounding boxes are currently transformed according to the skeleton layer. To use the tool, ensure that the skeleton layer is rendered natively in the left sidebar."
```